content/en/blog/2022/gc-elections-2022.md

```diff
@@ -0,0 +1,66 @@
+between 18 October 2022 and 20 October 2022, and the final election results will
+be announced 22 October 2022.
+
+## Vote!
+
+If you are a
+[member of standing](https://github.com/open-telemetry/community/blob/main/governance-charter.md#members-of-standing)
+in the OpenTelemetry community, we invite you to participate in this election to
+ensure that the community is well-represented in the Governance Committee. In
+this election four people must be elected, each with two-year terms.
+
+If you have made contributions to our ecosystem not measured by the automatic
+process, you can request
+[an exception](https://github.com/open-telemetry/community/blob/main/governance-charter.md#members-of-standing)
+to participate in the election before 23:59 UTC on 17 October 2022. The voter
+roll with all members of standing and approved exceptions will be published here
+by 11 October 2022 and continuously updated.
+
+Voting will be open between 00:00 UTC on 18 October 2022 00:00 UTC and 20
+October 2022 23:59 UTC on
+[Helios Voting](https://vote.heliosvoting.org/helios/elections/76558134-3384-11ed-8688-02871af94755/view);
+voters will need to sign in with their GitHub account.
+
+To learn more about the election process, read about all of the details
+[here](https://github.com/open-telemetry/community/blob/main/elections/2022/governance-committee-election.md).
+
+## Interested in joining the Governance Committee?
+
+If you’ve been working on OpenTelemetry and seeing it grow, or are an end-user
+that wants to help us make OpenTelemetry better, now’s the time to consider
+running for a seat on the Governance Committee. You can read about the
+Governance Committee's role in
+[this blog](https://medium.com/opentelemetry/opentelemetry-governance-committee-explained-860353baba0)
+post or refer to the
+[charter document](https://github.com/open-telemetry/community/blob/master/governance-charter.md).
+You may nominate yourself (or others!) by submitting a Pull Request against the
+[list of candidates](https://github.com/open-telemetry/community/blob/main/elections/2022/governance-committee-candidates.md)
+by 7 October 2022 23:59 UTC — more detailed requirements about the nomination
+and ratification process can be found
+[here](https://github.com/open-telemetry/community/blob/main/elections/2022/governance-committee-election.md).
+
+We would like to thank the GC members who have helped grow OpenTelemetry, and
+invite them to run for re-election if they so choose: Alolita Sharma, Daniel
+Dyla, Liz Fong-Jones and Morgan McLean. After this election, the Governance
+Committee will comprise 9 members with staggered 2-year terms, all elected by
+the community.
+
+## Questions?
+
+For any election related questions, please file an issue on the
+[community repo](https://github.com/open-telemetry/community/issues) here and
+tag @jpkrohling, @bogdandrutu, and @bhs. Or send us a message on CNCF Slack in
```

Comment:
Suggestion: Put links around the tags, e.g. `[@jpkrohling](https://github.com/jpkrohling)`